### PR TITLE
Add Feature Parameters to Count Permission Set Assignments

### PR DESCRIPTION
**Pull request by jjbennett**
- Added a new Feature Parameter that returns a count of how many Users are assigned the PMM: Deliver permission.
- Added a new Feature Parameter that returns a count of how many Users are assigned the PMM: Manage permission.
# Critical Changes

# Changes
-We updated PMM Feature Telemetry to include a count of Users assigned the PMM: Deliver permission, and a count of Users assigned the PMM: Manage permission. No personally identifiable information is collected through PMM Feature Telemetry. Read more about [PMM Feature Telemetry](https://powerofus.force.com/s/article/PMM-Feature-Telemetry).

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
~~Place holder data is incorporated (Refer to [Case Management Sample Data document](https://quip.com/cbFcAPJF8t0z))~~
~~Base axe-core JEST test included for any new LWC (No critical violations)~~
~~CRUD/FLS is enforced in Apex~~
~~Permission sets are updated to account for CRUD/FLS for new object metadata~~
- [x] All modified classes are unit tested (Apex) at 100% coverage
~~UX approval or UX not necessary~~
- [ ] PR contains draft release notes
- [ ] Attach work item to the pull request
- [ ] All **acceptance criteria** have been met
    - [ ] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by jjbennett** [comment]
[W-7844320](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008WMPkIAO/view)

File: force-app/main/default/classes/FeatureParameters.cls [hunk 1]
```diff
@@ -12,18 +12,28 @@ update the methods within the inner class accordingly
...
         '\'Active\'';
-
     private static final String CREATED_LAST_30_CONDITION = 'CreatedDate = LAST_N_DAYS:30';
+    private static final String PERMISSION_SET_NAME_CONDITION =
+        String.valueOf(PermissionSet.SObjectType) +
+        '.' +
+        String.valueOf(PermissionSet.Name) +
+        ' = ';
+    @TestVisible
+    private static final String DELIVER_PERMISSION = 'PMDM_Deliver';
```
Comment (microveldt) [inline force-app/main/default/classes/FeatureParameters.cls:34]: Maybe more of a question for the note on the PR, isn't the customer-facing permission PMM: Deliver and PMM: Manage?

Comment (jjbennett) [reply force-app/main/default/classes/FeatureParameters.cls:34]: Yes thanks!

File: force-app/main/default/classes/FeatureParameters_TEST.cls [hunk 2]
```diff
@@ -220,4 +220,106 @@ public with sharing class FeatureParameters_TEST {
...
+        Test.stopTest();
+
+        featureManagementStub.assertCalledWith(
+            'setPackageIntegerValue',
+            new List<Type>{ String.class, Integer.class },
+            new List<Object>{ expectedName, expectedValue }
+        );
+    }
+
+    ////////// Helper Methods //////////
```
Comment (bethbrains) [inline force-app/main/default/classes/FeatureParameters_TEST.cls:300]: Very clever!

**Comment by jjbennett** [comment]
Thanks for catching this one as well!

**Pull request merged**